src/pages/references/stacks-node-configuration.md

```diff
@@ -27,9 +27,9 @@ Example:
 stacks-node mocknet
 ```
 
-### krypton (deprecated)
+### krypton
```

Comment:
I thought we were going to stop using the noble gas names in public?

Reply:
We can, but moving away from _all_ the noble gas names in the docs is more complex than that. In certain parts of the docs like this one, `krypton` is syntactically the correct term to use because it corresponds directly to the blockchain code, configuration properties, or CLI subcommands available. So there would have to be code changes in the `stacks-node` before we're able to remove all references from the docs.

It's simple to remove the noble gas names in other explanative pieces of the docs though, like in the other comment referencing the `the public krypton regtest`.